deployment/helm/README.md

```diff
@@ -1,31 +1,64 @@
 # Deployment Guide
 
-- [Dependencies](#dependencies)
-- [Helm](#helm-repository)
-- [Quick Start](#quick-start)
-  - [Base install](#base-install)
-  - [Persistent Volumes](#persistent-volumes-and-persistent-volume-claims)
+- [Deployment Guide](#deployment-guide)
+  - [Dependencies](#dependencies)
+  - [Quick Start](#quick-start)
+    - [Helm Chart Repositories](#helm-chart-repositories)
+    - [Source Code Repository](#source-code-repository)
+    - [Quick Install](#quick-install)
+  - [Base Helm Chart](#base-helm-chart)
+    - [Development Environment](#development-environment)
+  - [Required Persistent Volumes](#required-persistent-volumes)
+  - [Required Persistent Volume Claim](#required-persistent-volume-claim)
+  - [IVOA Registry](#ivoa-registry)
   - [POSIX Mapper install](#posix-mapper-install)
-  - [Skaha install](#skaha-install)
-  - [Science Portal install](#science-portal-user-interface-install)
-  - [Cavern install](#cavern-user-storage-api-install)
-  - [Storage User Interface install](#user-storage-ui-installation)
-- [Obtaining a bearer token](#obtaining-a-bearer-token)
-- [Flow](#flow)
-- [Structure](#structure)
+    - [Skaha install](#skaha-install)
+    - [Science Portal User Interface install](#science-portal-user-interface-install)
+    - [Cavern (User Storage API) install](#cavern-user-storage-api-install)
```

Comment:
Under the declaration for `my-cavern-local-values-file.yaml`, please add an entry for a required `oidcURI` property to be set in the `deployment.cavern` section.